sw/host/opentitantool/src/command/image.rs

```diff
@@ -106,7 +102,11 @@ pub struct ManifestUpdateCommand {
         help = "Filename for an HJSON configuration specifying manifest fields"
     )]
     manifest: Option<PathBuf>,
-
+    #[structopt(
+        long,
+        help = "Filename for an HJSON configuration specifying manifest extension fields"
+    )]
+    manifest_ext: Option<PathBuf>,
```

Comment:
nit: do we have an example file that I can look at?

Reply:
Yes, [here](https://github.com/lowRISC/opentitan/blob/master/sw/host/opentitanlib/src/image/testdata/manifest_ext.hjson)